entity-types/uninstrumented-awslambdafunction/definition.yml

```diff
@@ -0,0 +1,6 @@
+domain: UNINSTRUMENTED
```

Comment:
Please remove this uninstrumented definition, due to: https://github.com/newrelic/entity-definitions/pull/1726/files#r1714853976